### ISSUE TITLE
Bigint error should have a specific error message

### ISSUE THREAD

**Issue by daniel7grant** [daniel7grant/eslint-plugin-typeorm-typescript#22]
Bigint has an unintuitive behaviour, as it is parsed to string on PostgreSQL and MySQL, because it could be out of range of the JavaScript number type (it's still a number in the SQLite driver). This means that throwing an error for this type of code is valid, however it seems a lot like a false positive.

```ts
@Column({ type: 'bigint' })
n: number;
```

There should be a specific lint message, that details the issue and explains why this should be a string.

**Comment by daniel7grant** [comment]
Closed by [v0.5.1](https://github.com/daniel7grant/eslint-plugin-typeorm-typescript/releases/tag/v0.5.1).
